tracer/src/Datadog.Trace/Ci/Configuration/CIVisibilitySettings.cs

```diff
@@ -25,6 +26,13 @@ public CIVisibilitySettings(IConfigurationSource source)
             ProxyNoProxy = proxyNoProxy.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
 
             TracerSettings = new TracerSettings(source);
+
+            if (Logs)
+            {
+                // Enable the direct log submission
+                TracerSettings.LogSubmissionSettings.DirectLogSubmissionEnabledIntegrations.Add("XUnit,Serilog,ILogger,Log4Net,NLog");
```

Comment:
I don't think we should enable the other loggers. They're not designed to be used together really, and you will get a lot of duplicate logs if you're using serilog + ilogger for example, which is common in aspnetcore. It's more standard to divert those logs to an xunit sink anyway, so I think that should be sufficient?

Reply:
makes sense, done in [55a25be](https://github.com/DataDog/dd-trace-dotnet/pull/2531/commits/55a25be642fdd90b8b33951ad8de1ad8ba87aba9)